bindings/zeebe/command/publish_message_test.go

```diff
@@ -76,8 +76,8 @@ func (cmd2 *mockPublishMessageCommandStep2) CorrelationKey(correlationKey string
 }
 
 // MessageId comes from the Zeebe client API and cannot be written as MessageID
-// Note that when the `stylecheck` linter is working again, this method will need "nolink:stylecheck" (can't change name to ID or it won't satisfy an interface)
-func (cmd3 *mockPublishMessageCommandStep3) MessageId(messageID string) commands.PublishMessageCommandStep3 {
+// Note that when the `stylecheck` linter is working again, this method will need "nolint:stylecheck" (can't change name to ID or it won't satisfy an interface)
```

Comment:
This line can be removed. Or changed to:

```
This must be named `MessageId` (with apologies to the linter 💄 #SorryNotSorry) or it won't satisfy the interface
```

Reply:
```suggestion
```